src/main/java/frc/robot/classes/spikemotor/SpikeMotor.java

```diff
@@ -0,0 +1,252 @@
+package frc.robot.classes.spikemotor;
+
+import java.util.ArrayList;
+
+/**
+* An abstracted motor interface designed for easy replacement.
+*/
+public abstract class SpikeMotor {
+    private boolean isInitialized = false;
+    private ArrayList<SpikeMotor> followers;
+    private SpikeMotor follower = null;
+
+    /**
+    * Initializes this motor with the given device number.
+    *
+    * @param deviceNumber    an int giving the device number
+    * @return                nothing
+    * @throws                IllegalStateException if already initialized
+    * @see                   #initImpl(int deviceNumber)
+    */
+    public final void init(int deviceNumber) {
+        if (isInitialized)
```

Comment:
I know this is valid code, but let's put this within {}'s to make it clearer:
```
if (isInitialized){
            throw new IllegalStateException(
                "Motor cannot be initialized twice"
            );
}
```

Reply:
Can do!